clusterloader2/testing/load/config.yaml

```diff
@@ -17,13 +17,13 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .ENABLE_JOBS false}}
-{{$ENABLE_PVS := DefaultParam .ENABLE_PVS false}}
```

Comment:
I think we disable PVs in kubemark tests, see https://github.com/kubernetes/perf-tests/issues/803
Because of that, we need to introduce a way to disable it in kubemark tests once we switch the default to true.
Sorry, this should've been also mentioned in the issue.

I think the easiest way to tackle this would be:
1. Sent a PR that will rename these variables to have CL2_ prefix, e.g. replace ENABLE_PVS to CL2_ENABLE_PVS. If a variable begins with CL2 it's possible to change it value by overriding env. Changing the name should be a no-op change because the name is not referenced anywhere outside this repo
2. Sent a PR to test-infra and change this file - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml by adding the following arg to all kubemark jobs (name contains kubemark) 
   ```
   - --env=CL2_ENABLE_PVS=false # TODO(https://github.com/kubernetes/perf-tests/issues/803): Fix me
   ```
3. Once you do it you should be good to go to change the default to false

Reply:
Nice, point 1 is done. 
Now we need to do point 2 and we'll be good to submit this PR